examples/python-cicd-with-cloudbuilder/.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea/
```

Comment:
Is there no generic .gitignore in the directories above this? If not, then this is fine.

Reply:
There is, I was hesitant to change global repo stuff, but if that's ok I can do that.

Reply:
You can make changes to the root .gitignore, but submit them as a separate PR. I'm not sure who would be the best reviewer, but if you can't track someone down LMK. Resolving this.